content/source/docs/cloud/registry/add.html.md

```diff
@@ -1,50 +1,54 @@
 description: |-
-  Learn to add public modules from the public Terraform Registry to your organization's private module registry.
+  Add providers and modules from the public Terraform Registry to your organization's private registry.
 ---
 
 [vcs]: ../vcs/index.html
 
-# Adding Public Modules to the Terraform Cloud Private Module Registry
+# Adding Public Providers and Modules to the Private Registry
 
-> **Hands-on:** Try the [Add Public Modules to your Private Module Registry](https://learn.hashicorp.com/tutorials/terraform/module-private-registry-add?in=terraform/modules&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial on HashiCorp Learn.
+You can add providers and modules from the public [Terraform Registry](/docs/registry/index.html) to your Terraform Cloud private registry. This lets you clearly designate which public providers and modules are recommended for the organization and makes their supporting documentation and examples centrally accessible.
 
-You can add modules from the public [Terraform Registry](/docs/registry/index.html) to your Terraform Cloud private module registry. Once added, Terraform Cloud automatically synchronizes public modules with their source on the Terraform Registry. The private module registry handles downloads and controls access with Terraform Cloud API tokens, so consumers don't need access to the module's source repository, even when they run Terraform from the command line.
+~> **Note:** Terraform Enterprise does not support public providers or modules.
+[permissions-citation]: #intentionally-unused---keep-for-maintainers
 
-All members of an organization can view and use public modules, but you need [owners team](/docs/cloud/users-teams-organizations/permissions.html#organization-owners) permissions to add them to the private module registry. Owners can add modules through the UI as shown below or through the [Registry Modules API](../api/modules.html#create-a-module-with-no-vcs-connection-).
+Once added, the private registry automatically synchronizes public providers and modules with their source on the Terraform Registry. The private registry handles downloads and controls access with Terraform Cloud API tokens, so consumers do not need access to the provider and module source repositories, even when they run Terraform from the command line.
 
--> **Note:** Public modules are not supported in Terraform Enterprise.
-[permissions-citation]: #intentionally-unused---keep-for-maintainers
+All members of an organization can view and use public providers and modules, but you need [owners team](/docs/cloud/users-teams-organizations/permissions.html#organization-owners) permissions to add them to the private registry. Owners can add providers and modules through the UI as detailed below or through the [Registry Providers API](/docs/cloud/api/providers.html) and the [Registry Modules API](../api/modules.html#create-a-module-with-no-vcs-connection-).
```

Comment:
```suggestion
All members of an organization can view and use public providers and modules, but only members of the [owners team](/docs/cloud/users-teams-organizations/permissions.html#organization-owners) can add them to the private registry. Owners can add providers and modules through the UI as detailed below or through the [Registry Providers API](/docs/cloud/api/providers.html) and the [Registry Modules API](../api/modules.html#create-a-module-with-no-vcs-connection-).
```

Reply:
it won't let me just apply the changes, so I'm making them locally and will push them up! So I'll resolve this but just know your changes have been applied!